Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1362,29 +1358,14 @@ inline std::optional<FontSelectionValue> BuilderConverter::convertFontStyleFromV
 
 inline FontSelectionValue BuilderConverter::convertFontWeight(BuilderState& builderState, const CSSValue& value)
 {
-    auto& primitiveValue = downcast<CSSPrimitiveValue>(value);
-    if (primitiveValue.isValueID()) {
-        auto valueID = primitiveValue.valueID();
-        if (valueID == CSSValueBolder)
-            return FontCascadeDescription::bolderWeight(builderState.parentStyle().fontDescription().weight());
-        if (valueID == CSSValueLighter)
-            return FontCascadeDescription::lighterWeight(builderState.parentStyle().fontDescription().weight());
-        if (CSSPropertyParserHelpers::isSystemFontShorthand(valueID))
-            return SystemFontDatabase::singleton().systemFontShorthandWeight(CSSPropertyParserHelpers::lowerFontShorthand(valueID));
```

Comment:
Does this cause a change in behavior? I assume we still want the system font weight, one way or another. I guess we'd need custom apply functions to do properly with the new setup.

Reply:
Good catch. That’s a mistake.

Reply:
It’s a mess because I’d have to make both a custom apply function and a custom initial function, even though I don’t really want either. The other option is to require the `isSystemFontShorthand` in all converters, and I think that’s what I’ll do.